doc/did-method-spec.md

```diff
@@ -399,6 +438,84 @@ The DID resolution result for a deactivated DID has the following shape:
 }
 ```
 
+## Metadata
+
+The `resolve` method returns an object with the following properties: `didDocument`, `didDocumentMetadata`,
+`didResolutionMetadata`.
+
+### DID Document Metadata
+
+When resolving a DID document that has had updates, the latest update MUST be listed in the `didDocumentMetadata`.
+* `versionId` MUST be the block height of the latest update.
+* `updated` MUST be the ISO date string of the block time of the latest update.
+
+Example:
+```json
+{
+  "didDocumentMetadata": {
+    "versionId": "12090175",
+    "updated": "2021-03-22T18:14:29Z"
+  }
+}
+```
+
+### DID Resolution Metadata
+
+```json
+{
+  "didResolutionMetadata": {
+    "contentType": "application/did+ld+json"
+  }
+}
+```
+
+## Resolving DID URIs with query parameters.
+
+### `versionId` query string parameter
+
+This DID method supports resolving previous versions of the DID document by specifying a `versionId` parameter.
+
+Example: `did:ethr:0x26bf14321004e770e7a8b080b7a526d8eed8b388?versionId=12090175`
+
+The `versionId` is the block height at which the DID resolution MUST be performed.
+Only ERC1056 events prior to or contained in this block height are to be considered when building the event history.
+
+If there are any events after that block that mutate the DID, the earliest of them SHOULD be used to populate the
+properties of the `didDocumentMetadata`:
+* `nextVersionId` MUST be the block height of the next update to the DID document.
+* `nextUpdate` MUST be the ISO date string of the block time of the next update.
```

Comment:
milliseconds dropped

Reply:
I added a comment about sub-second resolution and made #129 to fix the issue